types/passport-cognito/index.d.ts

```diff
@@ -21,8 +21,21 @@ export type CognitoVerifyFunction = (
   done: (error: any, user?: any) => void,
 ) => any;
 
+export type CognitoVerifyFunctionWithSession = (
+  accessToken: string,
+  idToken: string,
+  refreshToken: string,
+  user: object,
+	session: {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/passport-cognito/index.d.ts;
  Line: 29;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.